### DISCUSSION TITLE
Circle line weights

### DISCUSSION THREAD

**Discussion by kurt-rhee** [mozman/ezdxf#683] [Q&A]
Hello, 

I have been trying to change the line thickness of my circles, but have not yet been successful.  I have been trying to use the lineweight dxf attribute. Thickness also did not work.

![image](https://user-images.githubusercontent.com/33131958/167743780-c9a31bf4-73b1-44c5-88f1-25a48597d6b4.png)

        msp.add_circle(
            center=center,
            radius=r * 1.25,
            dxfattribs={
                'layer': layer,
                'true_color': ezdxf.colors.rgb2int(color),
                'lineweight': 211
            }
        )

Thank you for your help!

**Comment by mozman** [comment]
This seems to be an issue of the DXF viewer, `ezdxf` only sets the DXF attribute.

Lineweight for circles works in BricsCAD/TrueView:

![image](https://user-images.githubusercontent.com/4230966/167764172-6b6d97d0-fad3-4e1f-a071-f14bff4ebfb4.png)
